balena.yml

```diff
@@ -0,0 +1,14 @@
+version: "2"
+slug: "steamlink"
+name: "steamlink"
+type: "sw.application"
+assets:
+  - url: "https://github.com/Page-/steamlink"
+    name: "repository"
+  - url: "https://raw.githubusercontent.com/Page-/steamlink/master/logo.png"
+    name: "logo"
+data:
+  description: "Run steamlink on a Raspberry Pi and play!"
+  defaultDeviceType: "raspberrypi3"
+  supportedDeviceTypes:
+    - "raspberrypi3"
```

Comment:
It does support raspberrypi4-64, it's just it has to run 32bit code on it and there may be more rough edges (or less, I need to retest my rpi3 to be fair)